src/main/java/com/icthh/xm/ms/dashboard/web/rest/DefaultProfileResource.java

```diff
@@ -53,6 +54,7 @@ public DefaultProfileResource(DefaultProfileService defaultProfileService,
     @PostMapping("/default-profiles")
     @Timed
     @PreAuthorize("hasPermission({'defaultProfile': #defaultProfile}, 'DEFAULT_PROFILE.CREATE')")
+    @PrivilegeDescription("Privilege to create a new defaultProfile")
```

Comment:
replace defaultProfiles to more informative message. Describe what is defaultProfiles.